lib/rf/stylez/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Rf
   module Stylez
-    VERSION = "1.2.0"
```

Comment:
Version 1.2.0 never succeeded being pushed, so this "downgrade" is fine.
See: https://rubygems.org/gems/rf-stylez/